htdocs/inc.viewFolderTree.php

```diff
@@ -112,7 +112,9 @@
             $temp['type'] = "spotify";
             $titlefile = $subfolder."/title.txt";
             $coverfile = $subfolder."/cover.jpg";
-
+			
+		/* routine 1 (deprecated): getting spotify informations from mopidy API
```

Comment:
You could just remove the commented code, it's in git that should be sufficient.

Reply:
Done!
It's working fine on my phoniebox. I used it since 1,5 years now, beginning from version 1.1.6. Never had problems...